app/main/forms.py

```diff
@@ -2591,18 +2592,142 @@ class PostcodeForm(StripWhitespaceForm):
     than a city and the max side of alerting for non country size areas.
     """
     radius = GovukDecimalField(
-        "Add radius",
+        "Radius",
         param_extensions={
             "classes": "govuk-input govuk-input--width-5",
             "suffix": {"text": "km"},
             "attributes": {"pattern": "^-?\\d+(\\.\\d+)?$"},
         },
         validators=[
-            NumberRange(min=0.099, max=38.001, message="Enter a radius between 0.1km and 38.0km."),
-            DataRequired(message="Enter a radius."),
+            NumberRange(min=0.099, max=38.001, message="Enter a radius between 0.1km and 38.0km"),
+            DataRequired(message="Enter a radius"),
+            Only2DecimalPlaces(),
+        ],
+    )
+
+    def pre_validate(self, form):
+        return self.postcode.validate(form)
+
+    def post_validate(self):
+        return not self.errors and not self.postcode.errors and not self.radius.errors
+
+
+class DecimalCoordinatesForm(StripWhitespaceForm):
+    """
+    This form contains the input fields for creation of areas using decimal coordinates i.e. Latitude & longitude.
+    For the first coordinate, latitude, to be valid the value entered must be numeric and between -90 and 90,
+    hence the NumberRange validator.
+    For the second coordinate, longitude, to be valid the value again must be number and between -180 and 180.
+    There is additional validation post-submission of the form that checks whether the coordinate lies within
+    either the UK or the test polygons in Finland, and if both are false then an error is appended to the form.
+    """
+
```

Comment:
In the same way as EastingsNorthings I think I'd just call this LatitudeLongitudeCoordinatesForm as even if we do allow people to use other decimal based coordinate systems we'd proabably want them to be explicitly stated.